README.md

```diff
@@ -28,18 +28,18 @@ module "s3_site" {
 ![First Terraform Error](readme/terraform-apply-1.png)
 
 ## Inputs
-| Name                   | Type        | Description                                                                       | Default        |
-| ---------------------- | ----------- | --------------------------------------------------------------------------------- | -------------- |
-| hosted_zone_id         | string      | Hosted Zone ID                                                                    |
-| index_doc              | string      | The index document of the site.                                                   | index.html     |
-| error_doc              | string      | The error document (e.g. 404 page) of the site.                                   | index.html     |
-| origin_path            | string      | The path to the file in the S3 bucket (no trailing slash).                        | *Empty string* |
-| site_url               | string      | The URL for the site.                                                             |
-| wait_for_deployment    | string      | Define if Terraform should wait for the distribution to deploy before completing. | `true`         |
-| s3_bucket_name         | string      | Name of S3 bucket for the website                                                 |
-| tags                   | map(string) | A map of AWS Tags to attach to each resource created                              | {}             |
-| cloudfront_price_class | string      | The price class for the cloudfront distribution                                   | PriceClass_100 |
-| cors_rules             | list(object) | The CORS policies for S3 bucket                                                  | []             |
+| Name                   | Type         | Description                                                                       | Default        |
+| ---------------------- | ------------ | --------------------------------------------------------------------------------- | -------------- |
+| hosted_zone_id         | string       | Hosted Zone ID                                                                    |
+| index_doc              | string       | The index document of the site.                                                   | index.html     |
+| encryption_key_id      | string       | The AWS KMS master key ID used for the SSE-KMS encryption                         | 
```

Comment:
(nit) maybe say that the default is the default kms key?

Reply:
Technically the default is empty, but if you don't provide on then a `data` object will get the default key. You can't pass in an empty string or `aws/s3` because they aren't valid key IDs. Terraform does some magic behind the scenes to set the default key as `aws/s3`, but only if the parameter is omitted completely.

Reply:
Oh wait. I misread what you said. Good point.

Reply:
Done.